notebooks/Makefile

```diff
@@ -0,0 +1,9 @@
+TARGETS = testbook
```

Comment:
```suggestion
TARGETS = test-nb
```

Reply:
I saw this too and fixed it with the [commit](https://github.com/ufs-community/uwtools/pull/528/commits/4ad47cb44cc7100ae1877992bd01a42030f92161) just before this request.